requirements.txt

```diff
@@ -18,3 +18,4 @@ tabulate==0.8.5
 pyarrow==3.0.0
 stumpy==1.5.1
 torch>=1.3.1
+statsmodels>=0.12
```

Comment:
we already include statsmodels==0.11.1 (in line with WML) and I think all tests you use are included into that version

Reply:
done